platform/engineering/design-docs/2021-01-13-web-component-library.md

```diff
@@ -0,0 +1,221 @@
+# Web Component library
+
+**Author(s):** Brooks Johnson  
+**Last Updated:** January 13, 2021  
+**Status:** **Draft** | In Review | Approved  
+**Approvers:** _Chris Valarida_ [x], _Dror Matalon_ [ ], _Michael Fleet_ [ ]
+
+
+## Overview
+
+### Objective
+
+The goal is to establish a component library using the [Web Components standard](https://developer.mozilla.org/en-US/docs/Web/Web_Components). This will eventually supercede the current `component-library` code and some of the liquid HTML templates currently in `vets-website`. A non-goal is creating an entirely new visual experience as part of the component library.
```

Comment:
Why do we want a Web Components library? Please add one sentence in this paragraph, probably right after the first, summarizing the value of web components vs our existing approach.